server.go

```diff
@@ -288,7 +288,6 @@ func (srv *Server) ListenAndServe() error {
 	}
 	srv.stopUDP, srv.stopTCP = make(chan bool), make(chan bool)
 	srv.started = true
-	srv.lock.Unlock()
```

Comment:
Can't do that because it would leave it locked for the entirety of the serve lifetimes.